include/odp/api/spec/packet.h

```diff
@@ -1123,6 +1123,21 @@ int odp_packet_copy_data(odp_packet_t pkt, uint32_t dst_offset,
 int odp_packet_move_data(odp_packet_t pkt, uint32_t dst_offset,
 			 uint32_t src_offset, uint32_t len);
 
+/**
+ * Zero data within packet
+ *
+ * Zero 'len' bytes of data in the packet level offset. Maximum number of bytes
+ * to zero is packet data length. Packet is not modified on an error.
+ *
+ * @param pkt    Packet handle
+ * @param offset Byte offset into the packet
+ * @param len    Number of bytes to zero
+ *
+ * @retval 0 on success
+ * @retval <0 on failure
+ */
+int odp_packet_zero_data(odp_packet_t pkt, uint32_t offset, uint32_t len);
```

Comment:
I need it as a clear way to zero parts of packet for ICV calculation. It would be good to have inside crypto to make a step towards odp_crypto.c working with segmented packets.

Regarding `odp_packet_set_data()`. What usecase do you see for it with `value != 0`?